vnext/local-cli/runWindows/utils/msbuildtools.js

```diff
@@ -151,11 +144,21 @@ function checkMSBuildVersion(version, verbose) {
     console.log('Searching for MSBuild version ' + version);
   }
 
-  const requiresUWP =
-    'Microsoft.Component.MSBuild Microsoft.VisualStudio.ComponentGroup.UWP.VC.v141';
-  const vsPath = VSWhere(requiresUWP, version, 'installationPath');
+  // https://aka.ms/vs/workloads
+  const requires16 = [
+    'Microsoft.Component.MSBuild',
```

Comment:
This line is redundant. VC.Tools.* would implicitly add MSBuild.

Reply:
I'd rather be explicit than rely on implicit behavior